tests/feature/test_scenario_descriptions.py

```diff
@@ -0,0 +1,4 @@
+from pytest_bdd import scenarios
+
+# See feature file for reasons for executing (skipping) these tests
+scenarios("scenario_descriptions.feature")
```

Comment:
instead of testing it this way, can you test it in the way described in https://docs.pytest.org/en/latest/writing_plugins.html#testing-plugins? In this way we can better assert if the plugin itself works correctly, instead of "hoping" that it generates the correct test items.
This is an example of what I mean: https://github.com/pytest-dev/pytest-bdd/blob/386ed90cec68f4a9e000728928bbb4854f9a56d4/tests/feature/test_no_sctrict_gherkin.py#L4

Reply:
Also, I would not even have this test file, we can just squeeze it into tests/feature/test_description, as the feature we are testing is having a description in feature files.

Reply:
Thanks, I'll take a look and make that change.